src/agent/Makefile

```diff
@@ -159,7 +159,7 @@ vendor:
 
 #TARGET test: run cargo tests
 test: $(GENERATED_FILES)
-	@cargo test --all --target $(TRIPLE) $(EXTRA_RUSTFEATURES) -- --nocapture
+	@RUST_LIB_BACKTRACE=0 cargo test --all --target $(TRIPLE) $(EXTRA_RUSTFEATURES) -- --nocapture
```

Comment:
Commit description mentions `RUST_BACKTRACE=0`, here it is `RUST_LIB_BACKTRACE=0` and https://docs.rs/anyhow/latest/anyhow/ says :

```
If you want only panics to have backtraces, set RUST_BACKTRACE=1 and RUST_LIB_BACKTRACE=0
```

I suspect that description has a typo and should mention `RUST_LIB_BACKTRACE=0`, correct ?

I don't understand why `RUST_BACKTRACE=1` isn't explicitly passed to `cargo test` though. If it is set elsewhere, please provide an explanation in the commit description.

Also this is fixing a `make test` failure introduced by the previous commit IIUC. If so, you should either move this commit before or squash into the previous commit, for the sake of bisectability.

Reply:
Oh. This commit has been merged https://github.com/kata-containers/kata-containers/pull/9382/commits/1072658219279c7c55f520384c3cf240bf426d16. The message is actually a typo.

Let me added `RUST_BACKTRACE=1` explicitly and add a new commit for this.